draft-ietf-tls-tls13.md

```diff
@@ -2376,7 +2376,15 @@ RSASSA-PSS algorithms
   digest used in the mask generation function and the digest being signed are
   both the corresponding hash algorithm as defined in {{!SHS}}. When used in
   signed TLS handshake messages, the length of the salt MUST be equal to the
-  length of the digest output.  This codepoint is new in this document and is also
+  length of the digest output. When used in certificate signatures, the
+  length of the salt MUST be at least as long as the length of the
+  digest output, and the algorithm parameters MUST be DER encoded.
+  This codepoint indicates that the sender
+  will accept certificates in which the public key has OID rsaEncryption,
+  but not necessarily OID id-RSASSA-PSS {{RFC5756}}, whether with
```

Comment:
the "not necessarily" phrasing here is a little distracting for me.  It implies that maybe you could use a key with a PSS OID.  That's pretty awkward.

Reply:
I prefer here the #1089 wording:
```
 +: An RSA public key (OID rsaEncryption) MUST be supported, an RSASSA-PSS public
 +  key (OID id-RSASSA-PSS {{RFC5756}}) without parameters MUST be supported (such
 +  that any hash algorithm and salt length can be used), while an RSASSA-PSS
 +  public key (OID id-RSASSA-PSS) with parameters MAY be supported.
```

parameter-less rsa-pss is almost exactly the same as regular rsa, so adding support for it is not big change compared to the effort required to add TLS 1.3